### DISCUSSION TITLE
Add "Associated domain(s)" field

### DISCUSSION THREAD

**Discussion by WhamboMPS** [simple-login/app#590] [Feature Requests]
Email aliases are created for specific needs that are usually associated with one (or more than one) domains. For instance, I have created an alias with the prefix of `msft` on my custom domain (MSFT is the ticker symbol for Microsoft). I could conceivably use `msft@mydomain.com` when registering at `microsoft.com` but also at `hotmail.com`, `github.com`, `linkedin.com`, etc. since all these businesses/domains are owned by Microsoft. And there are definitely times when you need to answer the question _"What email alias did I use at domain xyz?"_ Therefore,

Please create an _Associated domain(s)_ field that exists in a many-to-one relationship with an Alias. This field should appear on the [New Custom Alias](https://app.simplelogin.io/dashboard/custom_alias) page (see #566) and should also appear on the dashboard cards. It is critical that this field be searchable from the freeform text field on the [Dashboard](https://app.simplelogin.io/dashboard/).

**Comment by ITechGeek81** [comment]
If labels and tags are added, you would probably be able to create tags for the domains.

https://github.com/simple-login/app/discussions/398
